### ISSUE TITLE
Review Comments

### ISSUE THREAD

**Issue by yogeshkumar05** [priyavit/starwars-redux#1]
https://github.com/priyavit/starwars-redux/blob/a16ee94863d166e867120ff35355589c606158ab/public/index.html#L10
Update title to something meaningful

**Comment by yogeshkumar05** [comment]
https://github.com/priyavit/starwars-redux/blob/starwar_redux/src/App.js#L15
Is binding necessary?

**Comment by yogeshkumar05** [comment]
https://github.com/priyavit/starwars-redux/blob/starwar_redux/src/App.js#L36
Can this data be handled through redux?

**Comment by yogeshkumar05** [comment]
https://github.com/priyavit/starwars-redux/blob/starwar_redux/src/App.js#L41
Please create a reusable Error Component to display error messages

**Comment by yogeshkumar05** [comment]
https://github.com/priyavit/starwars-redux/blob/starwar_redux/src/App.js#L61
Please create a reusable Text Component and use it

**Comment by yogeshkumar05** [comment]
https://github.com/priyavit/starwars-redux/blob/starwar_redux/src/App.js#L87
Use ES6 syntax wherever possible

**Comment by yogeshkumar05** [comment]
https://github.com/priyavit/starwars-redux/blob/starwar_redux/src/redux/loginaction.js#L7
create CONSTANTS and use from there

**Comment by yogeshkumar05** [comment]
https://github.com/priyavit/starwars-redux/blob/starwar_redux/src/starwar.scss#L15
create variables.scss

**Comment by yogeshkumar05** [comment]
https://github.com/priyavit/starwars-redux/blob/starwar_redux/src/planetDetails.js#L76
use propTypes and defaultProps

**Comment by yogeshkumar05** [comment]
https://github.com/priyavit/starwars-redux/blob/starwar_redux/src/displayPlanets.js#L5
covert to a function component

**Comment by priyavit** [comment]
> https://github.com/priyavit/starwars-redux/blob/starwar_redux/src/App.js#L15
> Is binding necessary?

Yes. If i don't bind with **this**, **this.setState** will result into undefined. If not in constructor then i will have to bind onclick.

**Comment by yogeshkumar05** [comment]
> > https://github.com/priyavit/starwars-redux/blob/starwar_redux/src/App.js#L15
> > Is binding necessary?
> 
> Yes. If i don't bind with **this**, **this.setState** will result into undefined. If not in constructor then i will have to bind onclick.

Please use arrow functions to avoid binding
